frontends/common/parser_options.cpp

```diff
@@ -223,6 +223,21 @@ ParserOptions::ParserOptions() : Util::Options(defaultMessage) {
             return true;
         },
         "[Compiler debugging] Folder where P4 programs are dumped\n");
+    registerOption(
```

Comment:
You said that this optimization causes some problems. If the problems are related to the existing open-source passes can you please file an issue describing the problems so we can address them?

Reply:
I have seen problems only in proprietary backend passes so far.

Reply:
But I haven't been testing open-source backends. I was relying on the existing CI tests in regards to open source backends.